lndclient/invoices_client.go

```diff
@@ -52,10 +52,10 @@ func (s *invoicesClient) WaitForFinished() {
 func (s *invoicesClient) SettleInvoice(ctx context.Context,
 	preimage lntypes.Preimage) error {
 
-	rpcCtx, cancel := context.WithTimeout(ctx, rpcTimeout)
+	_, cancel := context.WithTimeout(ctx, rpcTimeout)
 	defer cancel()
```

Comment:
This now looks useless. I think this timeout context needs to be passed into the `WithMacaroonAuth` call.

Reply:
Right, fixed it.